CONTRIBUTION.md

```diff
@@ -53,42 +41,102 @@ The format can be described more formally as follows:
 
 Follow-up to Id5e7cbb1.
 ```
 
-The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
-lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
-git tools.
-
-If it is a bug fix commit, the bug identifier should be mentioned in the commit message as `Fixes OCPBUGS-123` in a separate
-line.
-
-### PR workflow
-
-- Before submitting a PR
-  - Format your code and fix all the spelling/grammatical mistakes.
-  - Limit the column length of the code and the comments within the code to 120 characters.
-  - Error messages within the code should be limited to a single line.
-  - Update the documentation if your PR is introducing or changing user facing functionality.
-  - Ensure each PR commit compiles and all unit and e2e tests pass on your local machine.
-    - PRs that have vendor commits are an exception to this rule.
-- Following are the things you should keep in mind once you open a PR:
-  - Add a hold as soon as you open the PR by commenting `/hold`
-  - Wait for comments from at least 2 reviewers before pushing changes.
-  Open comments in the meanwhile can be worked on locally.
-  - PR comments should be addressed in new commits. Before final approval, they have to be squashed.
-  - If PR has multiple commits, changes requested should eventually be squashed into the original commit where the
-  change was requested before cancelling the hold. Each commit in the final PR before merge should pass the tests and be
-  usable. (see below)
-  - Once the PR is approved, remove the hold by commenting `/hold cancel` to merge the PR
+The subject should be no longer than 50 characters, and the body should be no longer than 80 characters. There are some
+[githooks](https://github.com/jorisroovers/gitlint) you can install that can help enforce these style requirements.
+
+## Opening a pull request
+
+Once you are done with the pre-PR checklist, push your changes and create a pull request!
+
+### Format of the pull request
+
+#### Title
+
+When creating your pull request, you should prefix it with the Jira issue, followed by the subsystem name in brackets. It
+should follow the format
+
+```WINC-<number>: [<subsystem>] <title>```
+
+For example, a docs PR would look like
+
+```WINC-959: [docs] reorganizes readme```
+
+In doing this, you’ll be linking your PR to the Jira ticket for tracking.
+
+In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
+
+- Opening the PR: `ASSIGNED` --> `POST`
+- PR merges: `POST` --> `MODIFIED`
+- `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
+- PR closes without merging, the Jira bug transitions back to NEW status.
+
+If you have no Jira issue or are not a Red Hat employee, your title should simply follow the format ```[<subsystem>] <title>```, with ```Fixes #<issue>``` in the comment. 
```

Comment:
Long line.

There is potential there could be PR that does not fixes any issue. So `Fixes #<issue>` is only when required.